integration_test/cases/repo.exs

```diff
@@ -222,6 +222,19 @@ defmodule Ecto.Integration.RepoTest do
     TestRepo.insert!(%User{inserted_at: datetime})
     assert [%{inserted_at: ^datetime}] = TestRepo.all(User)
   end
+  
+  @tag :returning
+  @tag :with_conflict_target
+  test "insert with returning with schema" do
```

Comment:
Please move the test to this describe group then and name it accordingly: https://github.com/alex88/ecto/blob/918d3db1c31bd8a194318323d4c3a22d4c4b30c5/integration_test/cases/repo.exs#L1086